lib/neo4j/railtie.rb

```diff
@@ -57,8 +58,8 @@ class Railtie < ::Rails::Railtie
       end
     end
 
-    def setup!(neo4j_config = nil)
-      type, url, path, options, wait_for_connection = final_config!(neo4j_config).values_at(:type, :path, :url, :options, :wait_for_connection)
+    def setup!(neo4j_config = OpenStruct.new(session: {}))
```

Comment:
This is a good change, but I wonder if we shouldn't use `ActiveSupport::OrderedOptions` instead of `OpenStruct` since this is the `Railtie`.  But we would need to refactor that rake task that we were talking about on Gitter